src/simplewallet/simplewallet.cpp

```diff
@@ -7024,13 +7026,13 @@ bool simple_wallet::lns_print_owners_to_names(const std::vector<std::string>& ar
         fail_msg_writer() << "arg too long, fails basic size sanity check max length = " << MAX_LEN << ", arg = " << arg;
         return false;
       }
-      if (!oxenmq::is_hex(arg))
-      {
-        fail_msg_writer() << "arg contains non-hex characters: " << arg;
-        return false;
-      }
+      //if (!oxenmq::is_hex(arg))
+      //{
+        //fail_msg_writer() << "arg contains non-hex characters: " << arg;
+        //return false;
+      //}
```

Comment:
I don't understand why this hex restriction was here.  Presumably it should be deleted rather than commented out, though?

Reply:
Or, possibly instead, we should be verifying:
- is hex chars *and* is 64 digits long
*or*
- is parseable as a valid wallet address`*`.

and fail if it matches neither of those.

---

`*` - Given an earlier comment I left also needed this: perhaps we need a "is_valid_address" somewhere, if we don't have it already?  I'd suggest: `bool is_valid_address(bool allow_subaddress = true, bool allow_integrated = true)` as the signature.